contact.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```

Comment:
Please go through some of the design inspiration on contact page since this is more of a signup/signin page